.circleci/config.yml

```diff
@@ -1,14 +1,194 @@
+            echo 'export AUTHENTICATION_SERVER_IMAGE="litmusportal-auth-server"' >> workspace/env-vars
+            echo 'export FRONTEND_IMAGE="litmusportal-frontend"' >> workspace/env-vars
+            echo 'export SUBSCRIBER_IMAGE="litmusportal-subscriber"' >> workspace/env-vars
+            echo 'export SELF_DEPLOYER_IMAGE="litmusportal-self-deployer"' >> workspace/env-vars
+            echo 'export IMGTAG="ci"' >> workspace/env-vars
+            cat workspace/env-vars >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Build graphql server docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${GRAPHQL_SERVER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/graphql-server
+      - run:
+          name: Save graphql server docker image
+          command: docker save -o workspace/${GRAPHQL_SERVER_IMAGE}.tar ${REPONAME}/${GRAPHQL_SERVER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build auth server docker image
+          command: docker images && docker build . -f Dockerfile -t ${REPONAME}/${AUTHENTICATION_SERVER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/auth
+      - run:
+          name: Save auth server docker image
+          command: docker save -o workspace/${AUTHENTICATION_SERVER_IMAGE}.tar ${REPONAME}/${AUTHENTICATION_SERVER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build subscriber docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${SUBSCRIBER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/subscriber
+      - run:
+          name: Save subscriber docker image
+          command: docker save -o workspace/${SUBSCRIBER_IMAGE}.tar ${REPONAME}/${SUBSCRIBER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build self-deployer docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${SELF_DEPLOYER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/tools/self-deployer
+      - run:
+          name: Save self-deployer docker image
+          command: docker save -o workspace/${SELF_DEPLOYER_IMAGE}.tar ${REPONAME}/${SELF_DEPLOYER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build frontend docker image
+          command: docker build . -f Dockerfile -t ${REPONAME}/${FRONTEND_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/frontend
+      - run:
+          name: Save frontend docker image
+          command: docker save -o workspace/${FRONTEND_IMAGE}.tar ${REPONAME}/${FRONTEND_IMAGE}:${IMGTAG}
+      - persist_to_workspace:
+          root: workspace
+          paths:
+            - litmusportal-server.tar
+            - litmusportal-subscriber.tar
+            - litmusportal-self-deployer.tar
+            - litmusportal-auth-server.tar
+            - litmusportal-frontend.tar
+            - env-vars
```

Comment:
@chandankumar4 We have to make it hardcoded here. The previous builds were failed due to this.

Only here, we have to make it. Other places are good to have env

Reply:
okay